### ISSUE TITLE
Documentation doesn't mention you can pass parameters

### ISSUE THREAD

**Issue by 2016rshah** [edgurgel/tentacat#72]
As you can see [here](https://github.com/edgurgel/tentacat/blob/master/lib/tentacat/repositories.ex#L25) the documentation doesn't mention you can pass in parameters like `[sort: "pushed"]` to the function even though that feature has already been implemented.

**Comment by edgurgel** [comment]
Thanks for the heads up. Feel free to send a PR adding this info :+1:

**Comment by 2016rshah** [comment]
There are multiple functions that pass parameters even if it isn't mentioned in the Tentacat docs (possible parameters are enumerated on the Github docs). Would it be better to add a blanket statement somewhere that mentions that if the Github documentation says you can pass parameters, then you can do so with Tentacat? That might be a bit more DRY than mentioning it in the documentation for every endpoint, right?

**Comment by edgurgel** [comment]
Agreed. We should probably set up an initial page for the docs and put a Getting Started explaining these things here: https://hexdocs.pm/tentacat
